### ISSUE TITLE
Scan mode that ignores known TTL points

### ISSUE THREAD

**Issue by baskettcase** [Noctem/Monocle#247]
Can we get a scan mode that will ignore known TTL points? This way we can finally get all those spawn points that seem to elude the scan. For example I have over 250 unknowns after several weeks. Maybe in that mode, once all are found it then moves to regular scanning?

**Comment by manolomanila** [comment]
Could this help?

```
GIVE_UP_KNOWN = 0
GIVE_UP_UNKNOWN = 3300
SKIP_SPAWN = 3300
```

**Comment by baskettcase** [comment]
Hmm.. I have no idea. Have you tried it? Maybe Ill try it tomorrow. @Noctem would this work?

**Comment by baskettcase** [comment]
Ok Im giving it a try today, hopefully Ill see those unknowns go down! :) Thanks for the advice!

**Comment by manolomanila** [comment]
I tried it some time ago and I think (didn't pay much attention sorry) it helped. Hope it does the trick for you!

I'm going to test it too tonight.

**Comment by baskettcase** [comment]
It does look like my know are 70 more than previously, but my unknown hasn't changed. That could be a bug though... It might indeed be working :)

**Comment by manolomanila** [comment]
I'm afraid it didn't seem to work. After 9 hours running with ~100 accounts, my Known went from **6600 to 6690**. Unknown stayed at 350. I've just restarted the scanner with the normal setting and have **6600 known again** and still 350 unknown. I don't really get what's happening. To begin with, if I run web.py and look at the map, it seems I have less than 100 red spawn points.

Just in case I'm missing something, I have more: 1500.

**Comment by baskettcase** [comment]
Yeah I did this all day yesterday and I went from 4892 known to 5038 known, but my unknown is still 265, it didn't budge at all.. so Yay for the new known.. but hmm.. buggy for the unknown?

**Comment by baskettcase** [comment]
@manolomanila Hey looks like the new scan when I did no-pickle it updated my unknowns and sure enough.. they are a whole lot less! Im at 88 unknown now, so yeah there does seem to be some sort of bug.
